### ISSUE TITLE
(Empty) XML tags should not be spanning multiple lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1137]
We should report (empty) XML elements that span multiple lines, such as in
```c#
/// Some text <see
///   cref="System.IO.FileStream"/> and some more text
```

Such elements should be placed on the same line, like
```c#
/// Some text <see cref="System.IO.FileStream"/>
/// and some more text
```

**Comment by RalfKoban** [comment]
proposed rule: MiKo_2233

**Comment by RalfKoban** [comment]
We should also consider non-empty XML tags (actually the start and end tags) to not span multiple lines.

**Comment by RalfKoban** [comment]
We should also consider attribute contents spanning multiple lines.
